libs/web3/src/lib/use-bridge-contract.ts

```diff
@@ -2,20 +2,22 @@ import type { Networks } from '@vegaprotocol/smart-contracts-sdk';
 import { VegaErc20Bridge } from '@vegaprotocol/smart-contracts-sdk';
 import { useWeb3React } from '@web3-react/core';
 import { useMemo } from 'react';
+import { useEnvironment } from '@vegaprotocol/react-helpers';
 
 export const useBridgeContract = () => {
+  const { VEGA_ENV } = useEnvironment();
   const { provider } = useWeb3React();
 
   const contract = useMemo(() => {
     if (!provider) {
       return null;
     }
     return new VegaErc20Bridge(
-      process.env['NX_VEGA_ENV'] as Networks,
+      VEGA_ENV as Networks,
```

Comment:
chore (non-blocking): can we do this case in the useEnviornment hook? As VEGA_ENV should always be a network